nodejs/week2/.gitignore

```diff
@@ -0,0 +1,46 @@
+# Logs
+*.log
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+
+# Runtime data
+pids
+*.pid
+*.seed
+*.pid.lock
+
+# Directory for instrumented libs generated by jscoverage/JSCover
+lib-cov
+
+# Coverage directory used by tools like istanbul
+coverage
+
+# nyc test coverage
+.nyc_output
+
+# Grunt intermediate storage (http://gruntjs.com/creating-plugins#storing-task-files)
+.grunt
+
+# Bower dependency directory (https://bower.io/)
+bower_components
+
+# Dependency directories
+node_modules/
+jspm_packages/
+
+# dotenv environment variables file
+.env
+
+# add gitignore for .vscode and .vscode/settings.json
+.vscode 
+.vscode/settings.json
+
+# React build
+build
+
+dist/
+
+.DS_Store
+
+package-lock.json
```

Comment:
It is a good idea to commit the package-lock.json. Using this, you can make sure that you have a service with a working dependency tree; https://docs.npmjs.com/cli/v9/configuring-npm/package-lock-json?v=true. 
It should be removed from this file.